db/system_keyspace.cc

```diff
@@ -262,6 +262,24 @@ schema_ptr raft_snapshots() {
     return schema;
 }
 
+schema_ptr repair() {
+    static thread_local auto schema = [] {
+        auto id = generate_legacy_id(NAME, REPAIR);
+        return schema_builder(NAME, REPAIR, std::optional(id))
+            .with_column("table_uuid", uuid_type, column_kind::partition_key)
+            .with_column("range_start", utf8_type, column_kind::partition_key)
+            .with_column("range_end", utf8_type, column_kind::partition_key)
```

Comment:
Perhaps a tuple<bigint, bigint> range. Why utf8_type?

Also should be a clustering key, so we can drop a table easily.

Reply:
I followed other places which uses utf8_type for range. 

Which columns should be the partition_key, only table_uuid?

Reply:
> I followed other places which uses utf8_type for range.

Let's not repeat mistakes.

 
> Which columns should be the partition_key, only table_uuid?

Yes, so that dropping a table can just drop that partition. Dropping a keyspace requires dropping each table individually anyway.

Reply:
This table is a local strategy table so a single partition per table is not a problem from the node distribution point of view. Is it problematic if we have lots of history per table, so we have giant partition.

Reply:
History cannot grow so large. Assuming by 2100 we will be extinct, we need about 80 years * 365 days * 256 ranges = 7.5M rows per table. That's not excessive. And of course we can TTL some of it.

Reply:
Good.